### DISCUSSION TITLE
Button press indication/ heating indication

### DISCUSSION THREAD

**Discussion by Ompapamowmow** [KlipperScreen/KlipperScreen#709] [Ideas]
It would be nice if there was a visual indication of a successful button press like an icon color change or flash or something like that. I often have trouble, especially on the macros page, knowing whether or not a touch was recognized so sometimes I end up pressing it again just to find out that it did work the first time. This goes the same for the pre heating buttons. The only way I can tell if the press was registered is to wait and see if the temperature is rising. It would be nice if you could add the set temperature to the display when the heaters are on. So when the heaters are off they would just display the current temp (ie 32°) and when the heaters are on display current and set temperature (ie 32°/200°).

**Comment by alfrix** [comment]
currently everything works as described, there is visual feedback and target temp is shown both in main and temp
i must disclaim that some gpio/spi screens on old OSs (buster) tend to not show the animations because they run on a very low fps (sometimes less than 20fps)
